src/GameBoard.java

```diff
@@ -1,32 +1,53 @@
+import sun.management.resources.agent;
+
 import java.util.ArrayList;
 import java.util.Stack;
 
 public class GameBoard {
 
-    ArrayList<Stack<Piece>> positions;
+    ArrayList<Stack<Piece>> positions = new ArrayList<>();
     Agent white;
     Agent black;
 
     int blackPiecesInGame = 15;
     int whitePiecesInGame = 15;
 
-    public GameBoard(ArrayList<Stack<Piece>> positions, Agent white, Agent black){
-        this.positions = positions;
+    public GameBoard(Agent white, Agent black){
         this.white = white;
         this.black = black;
+        for(int i = 0; i<27; i++){
```

Comment:
Wie genau war nochmal die Aufteilung für das Board aufs array? Hatten wir nicht gesagt -1 und 0 als Sonderfelder, dann 1 bis 24 normal und wieder 25 und 26 als Sonderfelder? Müsste die for-Schleife dann bei -1 anfangen?